scripts/local_testnet/README.md

```diff
@@ -5,58 +5,105 @@ This setup can be useful for testing and development.
+
+```bash
+cd geth-alltools-linux-amd64-1.13.1-3f40e65c
+sudo cp geth bootnode /usr/local/bin
+```
+
+After that We can remove the downloaded files:
+
+```bash
+cd ~
+rm -r geth-alltools-linux-amd64-1.13.1-3f40e65c geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
+```
+
+We are now ready to start a local testnet.
+
 ## Starting the testnet
 
-Modify `vars.env` as desired.
+To start a testnet using the predetermined settings:
+
+```bash
+cd ~
+cd /lighthouse/scripts/local_testnet
+./start_local_testnet.sh genesis.json
+```
+
+This will execute the script and if the testnet setup is successful, you will see "Started!" at the end. 
 
 The testnet starts with a post-merge genesis state. 
-Start a consensus layer and execution layer boot node along with `BN_COUNT`
-number of beacon nodes each connected to a geth execution client and `VC_COUNT` validator clients.
+The testnet starts a consensus layer and execution layer boot node along with `BN_COUNT`
+(the number of beacon nodes) each connected to a geth execution client and `VC_COUNT` (the number of validator clients). By default, `BN_COUNT=4`, `VC_COUNT=4`. 
 
 The `start_local_testnet.sh` script takes four options `-v VC_COUNT`, `-d DEBUG_LEVEL`, `-p` to enable builder proposals and `-h` for help. It also takes a mandatory `GENESIS_FILE` for initialising geth's state.
 A sample `genesis.json` is provided in this directory.
 
-The `ETH1_BLOCK_HASH` environment variable is set to the block_hash of the genesis execution layer block which depends on the contents of `genesis.json`. Users of these scripts need to ensure that the `ETH1_BLOCK_HASH` variable is updated if genesis file is modified.
-
 The options may be in any order or absent in which case they take the default value specified.
 - VC_COUNT: the number of validator clients to create, default: `BN_COUNT`
 - DEBUG_LEVEL: one of { error, warn, info, debug, trace }, default: `info`
 
+The `ETH1_BLOCK_HASH` environment variable is set to the block_hash of the genesis execution layer block which depends on the contents of `genesis.json`. Users of these scripts need to ensure that the `ETH1_BLOCK_HASH` variable is updated if genesis file is modified.
 
+To view the beacon, validator client and geth logs:
 
 ```bash
-./start_local_testnet.sh genesis.json
+tail -f ~/.lighthouse/local-testnet/testnet/beacon_node_1.log
+taif -f ~/.lighthouse/local-testnet/testnet/validator_node_1.log
+tail -f ~/.lighthouse/local-testnet/testnet/geth_1.log
 ```
 
+where `beacon_node_1` can be changed to `beacon_node_2`, `beacon_node_3` or `beacon_node_4` to view logs for different beacon nodes. The same applies to validator clients and geth nodes. 
+
 ## Stopping the testnet
 
-This is not necessary before `start_local_testnet.sh` as it invokes `stop_local_testnet.sh` automatically.
+To stop the testnet, navigate to the directory `cd /lighthouse/scripts/local_testnet`, then run the command:
```

Comment:
Same as above, I think it's usually fine to refer to files using relative path to the repo

```suggestion
To stop the testnet, navigate to the directory `scripts/local_testnet`, then run the command:
```

or full repo if you prefer (but without the `/` at the beginning)

```suggestion
To stop the testnet, navigate to the directory `cd lighthouse/scripts/local_testnet`, then run the command:
```

Reply:
Fix in [9b4a969](https://github.com/sigp/lighthouse/pull/4749/commits/9b4a9699420ca28bf43d1ea588d448249f3128ae)  using `~` just in case the terminal is in a different working directory